crates/rooch-framework/sources/bitcoin_network.move

```diff
@@ -0,0 +1,91 @@
+module rooch_framework::bitcoin_network{
+    use std::error;
+    use std::string::{Self, String};
+
+    const ErrorUnknownNetwork: u64 = 1;
+
+    ///Currently, Move does not support enum types, so we use constants to represent the network type.    
+    /// Mainnet Bitcoin.
+    const NETWORK_BITCOIN: u8 = 1;
+    public fun network_bitcoin(): u8 {
+        NETWORK_BITCOIN
+    }
+
+    /// Bitcoin's testnet network.
+    const NETWORK_TESTNET: u8 = 2;
```

Comment:
Shall we rename it to NETWORK_BITCOIN_TESTNET for clarity?

Reply:
Read the latest code. it in a bitcoin-move framework.